app/javascript/shared/components/__tests__/tags.test.jsx

```diff
@@ -21,6 +24,10 @@ describe('<Tags />', () => {
 
   describe('handleKeyDown', () => {
     it('does not call preventDefault on used keyCode', () => {
+      // Only one call is being made to /tags/suggest when the comma key is pressed
+      // It didn't seem worth it to inspect the whole mock object to ensure the right URL etc.
+      fetch.mockResponseOnce('[]');
```

Comment:
I'm returning an empty array of tags so the fetch doesn't break, but this result is not used at all.